### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by sahilyadav902**
Added instructions for PersonApp

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,18 +132,20 @@ naval_chat_bot.add("web_page", "https://nav.al/agi")
 
 # Embed Local Resources
 naval_chat_bot.add_local("qna_pair", ("Who is Naval Ravikant?", "Naval Ravikant is an Indian-American entrepreneur and investor."))
-```
+````
```
Comment (cachho) [inline README.md:135]: 1 too many `

Comment (cachho) [reply README.md:135]: you should also specify python here for better visual design

File: README.md [hunk 2]
```diff
@@ -303,40 +324,44 @@ print(naval_chat_bot.query("What unique capacity does Naval argue humans possess
...
-*coming soon*
+_coming soon_
 
 #### **Query Config**
 
-|option|description|type|default|
-|---|---|---|---|
-|template|custom template for prompt|Template|Template("Use the following pieces of context to answer the query at the end. If you don't know the answer, just say that you don't know, don't try to make up an answer. \$context Query: $query Helpful Answer:")|
+| option   | description                | type     | default                                                                                                                                                                                                             |
+| -------- | -------------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```
Comment (cachho) [inline README.md:344]: this is hard to maintain, and looks weird in smaller windows. I'd ditch it. But it's up to you.

File: README.md [hunk 3]
```diff
@@ -303,40 +325,44 @@ print(naval_chat_bot.query("What unique capacity does Naval argue humans possess
...
 ### Configs
+
 This section describes all possible config options.
 
 #### **InitConfig**
-|option|description|type|default|
-|---|---|---|---|
-|ef|embedding function|chromadb.utils.embedding_functions|{text-embedding-ada-002}|
-|db|vector database (experimental)|BaseVectorDB|ChromaDB|
+
```
Comment (taranjeet) [inline README.md:332]: we can skip this part for this PR.

File: README.md [hunk 4]
```diff
@@ -303,40 +325,44 @@ print(naval_chat_bot.query("What unique capacity does Naval argue humans possess
...
+| db     | vector database (experimental) | BaseVectorDB                       | ChromaDB                 |
 
 #### **Add Config**
 
-*coming soon*
+_coming soon_
 
 #### **Query Config**
 
-|option|description|type|default|
```
Comment (taranjeet) [inline README.md:320]: same we can skip this part for this PR.

**Review by taranjeet — APPROVED** [review]

**Pull request merged**